joomla/joomla-cms
# [4.0] Fix template preview JS error

Pull Request for Issue #20492.

### Summary of Changes
Listing pages allow for multi-select except on the Templates page. This PR prevents the JS error when there are no checkboxes for multi-select.

### Testing Instructions
install 4.0-dev
login to admin
go to template manager
click templates
click the image

See #20492 for video recording of error.

### Expected result
No JS error


### Actual result
JS error

> multiselect.min.js?132339e72563fafc2a9b345f445e4d39:1 Uncaught TypeError: Cannot read property 'checked' of null


### Documentation Changes Required
none
